source/_docs/autostart/systemd.markdown

```diff
@@ -35,7 +36,7 @@ WantedBy=multi-user.target
 
 ### Python virtual environment
 
-If you've setup Home Assistant in `virtualenv` following our [Python installation guide](/getting-started/installation-virtualenv/) or [manual installation guide for Raspberry Pi](/getting-started/installation-raspberry-pi/), the following template should work for you. If Home Assistant install is not located at `/srv/homeassistant`, please modify the `ExecStart=` line appropriately. `YOUR_USER` should be replaced by the user account that Home Assistant will run as (e.g `homeassistant`).
+If you've setup Home Assistant in `virtualenv`, the following template should work for you.If Home Assistant install is not located at `/srv/homeassistant`, please modify the `ExecStart=` line appropriately. `YOUR_USER` should be replaced by the user account that Home Assistant will run as (e.g `homeassistant`).
```

Comment:
Not sure why you deleted the space.

Only the Raspberry Pi guide gives the location of `/srv/homeassistant`, so this edit could cause confusion. Those following the first guide will likely have installed it in their home directory.

Reply:
Sounds good. Don’t know what happened to the space.